infrastructure/src/main/java/org/corfudb/infrastructure/management/FailureDetector.java

```diff
@@ -201,33 +174,6 @@ private PollReport pollRound(
                 .build();
     }
 
-    /**
-     * Tune timeouts after each poll iteration, according to list of failed and connected nodes
-     *
-     * @param router     client router
-     * @param currReport current poll report
-     * @return updated poll interval
-     */
-    private long modifyIterationTimeouts(Map<String, IClientRouter> router, PollReport currReport) {
-        long pollIterationStartTime = System.nanoTime();
-        long pollInterval = initialPollInterval;
-
-        Set<String> allReachableNodes = currReport.getAllReachableNodes();
-
-        if (!currReport.getFailedNodes().isEmpty()) {
-            // Increase the inter iteration sleep time by increasing poll interval if the
-            // period time increases
-            long iterationElapsedTime = TimeUnit.MILLISECONDS.convert(
```

Comment:
This increased poll iteration time is lost in the new code refactoring. Please restore this.

Reply:
fixed